support/mender-inventory-geo

```diff
@@ -31,15 +31,17 @@ err() {
  exit $rc
 }
 
+TIMEOUT=10
```

Comment:
can we add _s here to indicate that it is in seconds?

Reply:
WE can but that makes using it more difficult since we have to strip off the s character.  I'll just add a comment that the time is in seconds.